assets/sass/components/settings/_googlesitekit-settings-module.scss

```diff
@@ -127,6 +127,16 @@
 		margin: 0 0 8px 0;
 	}
 
+	.googlesitekit-settings-module__fields-group {
+		border-bottom: 1px solid $c-mercury;
+		margin-bottom: 1em;
+
+		&:last-child {
+			border-bottom: none;
+			margin-bottom: 0;
+		}
+	}
```

Comment:
This is probably more related to a problem in the component code, but I see two Storybook stories where there is still an undesired double border now:
* https://google.github.io/site-kit-wp/storybook/pull/4229/?path=/story/analytics-module-settings--existing-tag-w-access-gtm-property-w-access
* https://google.github.io/site-kit-wp/storybook/pull/4229/?path=/story/analytics-module-settings--existing-tag-w-access-gtm-property-w-o-access

Could you check those?

Reply:
Ok, I think it should be good now.